community/event_management/doctype/exhibitor_registration/exhibitor_registration.json

```diff
@@ -0,0 +1,55 @@
+{
+ "actions": [],
+ "autoname": "Exhibitor-Regis-.####.",
+ "creation": "2021-08-16 16:26:46.189119",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "full_name",
+  "company",
+  "logo"
+ ],
+ "fields": [
+  {
+   "fieldname": "full_name",
```

Comment:
Link user doctype here. Full Name and Logo will come from user doctype. The company field can be here along with the URL to their website. The design also has a company field